lib/ChartBase.js

```diff
@@ -22,7 +23,8 @@ const legendIface = {
 
   textColor: PropTypes.number,
   textSize: PropTypes.number,
-  fontFamily: PropTypes.string,
```

Comment:
nope it was just a place holder for the developer to fill in on the native sides